pennylane/ops/functions/is_commuting.py

```diff
@@ -42,24 +36,29 @@ def _pword_is_commuting(pauli_word_1, pauli_word_2, wire_map=None):
     **Example**
 
     >>> wire_map = {'a' : 0, 'b' : 1, 'c' : 2}
-    >>> pauli_word_1 = qml.X('a') @ qml.Y('b')
-    >>> pauli_word_2 = qml.Z('a') @ qml.Z('c')
-    >>> is_commuting(pauli_word_1, pauli_word_2, wire_map=wire_map)
+    >>> pauli_word_1 = qml.prod(qml.X("a"), qml.Y("b"))
+    >>> pauli_word_2 = qml.prod(qml.Z("a"), qml.Z("c"))
+    >>> _pword_is_commuting(pauli_word_1, pauli_word_2, wire_map=wire_map)
     False
+
+    >>> wire_map = {'a' : 0, 'b' : 1, 'c' : 2}
+    >>> pauli_word_1 = qml.sum(qml.X('a') , qml.Y('b'))
+    >>> pauli_word_2 = qml.sum(qml.Z('c') , qml.X('a'))
+    >>> _pword_is_commuting(pauli_word_1, pauli_word_2, wire_map=wire_map)
+    True
     """
 
     if wire_map is None:
         wire_map = _wire_map_from_pauli_pair(pauli_word_1, pauli_word_2)
+        pauli_word_1 = pauli_word_1.map_wires(wire_map)
+        pauli_word_2 = pauli_word_2.map_wires(wire_map)
```

Comment:
Can we just get rid of this now?

Reply:
So we get rid of the `wire_map` argument that the `is_commuting` function currently accepts?